<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Блокнот</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Поиск</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QMdiArea" name="mdiArea"/>
        <widget class="fileManager" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>606</height>
          </size>
         </property>
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="searchView">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Язык</string>
     </property>
     <addaction name="russian"/>
     <addaction name="english"/>
    </widget>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Выбрать тему</string>
     </property>
     <addaction name="light"/>
     <addaction name="dark"/>
    </widget>
    <addaction name="open_2"/>
    <addaction name="newDoc"/>
    <addaction name="readOnly"/>
    <addaction name="separator"/>
    <addaction name="save"/>
    <addaction name="printDoc"/>
    <addaction name="separator"/>
    <addaction name="menu_3"/>
    <addaction name="separator"/>
    <addaction name="keysetting"/>
    <addaction name="menu_2"/>
    <addaction name="help"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Работа с текстом</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Выравнивание</string>
     </property>
     <addaction name="onLeft"/>
     <addaction name="onCenter"/>
     <addaction name="onRight"/>
    </widget>
    <addaction name="copyFT"/>
    <addaction name="setFT"/>
    <addaction name="separator"/>
    <addaction name="menu_5"/>
    <addaction name="separator"/>
    <addaction name="font"/>
    <addaction name="separator"/>
    <addaction name="dateTimeAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="open_2">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="help">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="russian">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="english">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Английский</string>
   </property>
  </action>
  <action name="keysetting">
   <property name="text">
    <string>Настройки раскладки клавиатуры</string>
   </property>
  </action>
  <action name="readOnly">
   <property name="text">
    <string>Открыть в режиме чтения</string>
   </property>
  </action>
  <action name="light">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Светлая</string>
   </property>
  </action>
  <action name="dark">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Тёмная</string>
   </property>
  </action>
  <action name="newDoc">
   <property name="text">
    <string>Открыть новое окно</string>
   </property>
  </action>
  <action name="printDoc">
   <property name="text">
    <string>Печать</string>
   </property>
  </action>
  <action name="copyFT">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Копировать формат</string>
   </property>
  </action>
  <action name="setFT">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Применить формат</string>
   </property>
  </action>
  <action name="onLeft">
   <property name="text">
    <string>Слева</string>
   </property>
  </action>
  <action name="onCenter">
   <property name="text">
    <string>Посередине</string>
   </property>
  </action>
  <action name="onRight">
   <property name="text">
    <string>Справа</string>
   </property>
  </action>
  <action name="font">
   <property name="text">
    <string>Шрифт</string>
   </property>
  </action>
  <action name="dateTimeAction">
   <property name="text">
    <string>Вставить время </string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>fileManager</class>
   <extends>QWidget</extends>
   <header>filemanager.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
